about/index.md

```diff
@@ -19,33 +23,40 @@ Structured Threat Information Expression (STIX™) is a structured language to r
+Individuals and organizations are invited to join the STIX community!
+
+Feature requests and bugfixes are managed using [open source repositories on Github](https://github.com/stixproject/schemas)
 
 - [Mailing List](https://stix.mitre.org/community/registration.html) — where community members discuss the latest drafts of the STIX schemas, specifications, utilities, technical documents, and other items integral to the ongoing development of STIX.
 - [Chat](https://gitter.im/STIXProject/schemas) - informal discussions and help
-- [Code Repositories](https://github.com/STIXProject/) — the central location for STIX Community members to make open-source contributions to STIX development and manage issue tracking for the STIX schemas, tools, specifications, and other supporting information and items.
+- [Code Repositories](https://github.com/STIXProject/) — Github organization for development of schemas, tools, specifications, and other supporting information and items.
 
 ## Frequently Asked Questions
 
-#### Who is STIX for? What does STIX do for me?
+## Who is STIX for? What does STIX do for me?
+
+STIX is for anyone involved in defending networks or systems against cyber threats, including cyber defenders, cyber threat analysts, malware analysts, security tool vendors, security researchers, threat sharing communities, and more. 
 
-STIX is for anyone involved in defending networks or systems against cyber threats, including cyber defenders, cyber threat analysts, malware analysts, security tool vendors, security researchers, threat sharing communities, and more. STIX provides a common language for describing cyber threat information so it can be shared, stored, and otherwise used in a consistent manner that facilitates automation.
 
-#### How do I get it?
+## How do I get it?
 
 The STIX Language is available on the [STIX website](https://stix.mitre.org/language/).
 
 Bindings and related tools to help process and work with STIX are [open source on Github](https://github.com/STIXProject).
 
-#### Where can I find examples of STIX data? Are there any STIX repositories?
+## Where can I find examples of STIX data? Are there any STIX repositories?
+
+Certain STIX repositories [are available](/adopters) are available for free and for-pay.
 
-The STIX Samples page on this website hosts full threat reports expressed via STIX, including Mandiant's APT1 report and FireEye's Poison Ivy report. [Idioms](/idioms) also provide good constrained examples.
+[Example threat reports](/examples) include STIX versions of Mandiant's APT1 report and FireEye's Poison Ivy report. 
 
-In addition to the MITRE samples, community members have set up [TAXII](https://taxii.mitre.org) repositories containing STIX content and even directories pointing to those repositories. One example repository is http://hailataxii.com.
```

Comment:
IMO we should continue linking to hailataxii. People ask for STIX data all the time, it's the best source for that, and it's free.